packages/core/src/Modules/Cashier/Components/Form/account-transfer-form.jsx

```diff
@@ -46,7 +46,12 @@ const AccountOption = ({ account, idx }) => {
             </div>
 
             <span className='account-transfer__balance'>
-                <Money amount={account.balance} currency={account.currency} show_currency />
+                <Money
+                    amount={account.balance}
+                    currency={account.currency}
+                    has_sign={account.balance < 0}
+                    show_currency
+                />
```

Comment:
Just a Question, should it be `-$1.00` instead of `$-1.00`?

Reply:
Interesting question 😄, It should be `-1.00 USD` I don't think we use symbols here. [This](https://github.com/binary-com/deriv-app/blob/3536ada085ffa546d6f4c80ef1f36fb5c35b3d62/packages/trader/src/App/Components/Elements/__tests__/money.spec.js) explains everything about `Money` component

Reply:
`currency_icon` is passed outside of this component, so if we want to change the order for negative values, we'll need to add icon to `Money` and remove it from outside components